apps/system/test/marionette/lib/video_player.js

```diff
@@ -58,13 +58,16 @@ VideoPlayer.prototype = {
   },
 
   invokeControls: function() {
-    if (!this.visibleControls()) {
-      this.rootElement.tap();
-
+    if(this.visibleControls()) {
+      this.rootElement.tap(0, 0);
```

Comment:
It's to enforce them to disappear ASAP - it turned out to have some influence on the test outcome.
However: I'm fine removing it to see if it has some actual influence.